api/authenticator.py

```diff
@@ -158,7 +157,20 @@ def __repr__(self):
             self.permanent_id, self.authorization_identifier,
             self.username
         )
-        
+
+    @hybrid_property
+    def fines(self):
+        return self._fines
+
+    @fines.setter
+    def _set_fines(self, value):
+        """When setting patron fines, only store the numeric portion of 
+        a Money object.
+        """
+        if isinstance(value, Money):
+            value = value.amount
+        self._fines = value
```

Comment:
Is there ever a difference in currency?  Like, do some libraries count in pennies, while others in dollars?